script/results.html

```diff
@@ -4,44 +4,123 @@
 <head>
   <meta charset=utf-8>
   <title>TNF claim.json parser</title>
+  <meta charset="utf-8">
   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
     integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+
+  <style type="text/css">
+    li.dropdown-menu {
+      background-color: white;
+    }
+
+    table {
+      border-collapse: collapse;
+    }
+
+    table td {
+      border: 1px solid #000;
+    }
+    tg {
+      background-color: #28a745;
+    }
+
+    tred {
+	    background-color : #dc3545;
+    }
+
+    ty {
+	    background-color : #ffc107;
+    }
+
+    tgy {
+	    background-color : 	#6c757d;
+    }
+  </style>
 </head>
 
 <body>
   <nav class="navbar navbar-dark bg-dark">
     <a class="navbar-brand" href="#">Test Network Function result parser</a>
-    <form class="form-inline">
-      <div class="mb-3">
-        <label for="formFile" class="form-label text-light">Upload claim.json file</label>
-        <input class="form-control" type="file" id="formFile">
-      </div>
-    </form>
+    <div class="mb-3">
+      <label for="formFile" class="form-label text-light">Upload claim.json file</label>
+      <input class="form-control" type="file" id="formFile">
+      <label for="selectBox" class="form-label text-light">
+        Choose a scenario:</label>
+      <select name="class" id="selectBox" onchange="changeFunc()">
+        <option value="all"> All</option>
+        <option value="faredge"> Far Edge</option>
+        <option value="telco"> Telco</option>
+        <option value="nontelco"> Non Telco</option>
+      </select>
+      <table>
+        <tr>
+          <td>
+            <tred>Failed Madatory</tred>
```

Comment:
Remove the legend from top of the page and just color the Test summary text with the corresponding color:
Total: 48
Passed: 33
Skipped: 14
Failed: 1